plugins/node/plugin.zsh

```diff
@@ -8,11 +8,17 @@ GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VE
 geometry_prompt_node_setup() {}
 
 geometry_prompt_node_render() {
-    if (( $+commands[node] )); then
-        GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
-        GEOMETRY_NPM_VERSION="$(npm --version 2> /dev/null)" 
-        echo "$GEOMETRY_NODE_NPM_VERSION $GEOMETRY_NODE_VERSION ($GEOMETRY_NPM_VERSION)"
+    local -a files
+    files=($(ls -a ${PWD} | egrep -e ".nvmrc|yarn.lock"))
+
+    if [[ ${#files[@]} -gt 0 ]]; then
+        if (( $+commands[node] )); then
+            GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
+            GEOMETRY_NPM_VERSION="$(npm --version 2> /dev/null)" 
```

Comment:
Now, here is the thing. What we want to do is to:

1 - Check if we're in a node project
2 - Check which package manager the project is using (either `npm` or `yarn`)

With point 1) we are trying to avoid displaying node/npm/yarn version unless it's needed at all (ie, when the user is sitting in a node project).
We can achieve this by looking verifying the existence of `package.json`, which conventionally is used by `npm` and by `yarn`.  This check alone solves point 1):

    if [[ ! -f "$PWD/package.json" ]]; then # Check we're in a node project
        return # Don't print anything when we're not...
    fi

In regard of point 2) we'll need to check for a `yarn.lock` file and the existence of `yarn` command. Why? Well, a `yarn.lock` will tell us the project uses `yarn` rather than `npm` (both of them use `package.json`, so it doesn't tell us much). We also need to check for `yarn` command because the user may checkout a project that is using `yarn` but he may not have it installed yet (`npm` comes with `node` installation, so in this case there is no need to check it).
So, all in all it may look as follows:

    local packager=npm # Use npm by default (comes with node installation)
    if [[ $+commands[yarn] && -f "$PWD/yarn.lock" ]]; then # Check yarn and yarn.lock exists
        packager=yarn # If we're in a project using yarn and yarn command is available, use it instead of npm
    fi
    NPM_YARN_VERSION=$($packager --version 2> /dev/null) # Display either npm or yarn version

In the line above we're using `packager` variable (which is a string, either "npm" or "yarn") to dynamically form a command with one of these two strings, depending on the condition above, it may execute `npm --version 2> /dev/null` or `yarn --version 2> /dev/null`.

And that's all, we then will display `node` version and `npm/yarn` version.

It does look like a lot of things to do but we're almost there :)